### PR TITLE
Immutable diamond

### PR DESCRIPTION
**Pull request by 0xDEnYO**

### CODE REVIEW

File: deployments/goerli.diamond.staging.json [hunk 1]
```diff
@@ -1,12 +1,5 @@
...
   "facets": [
-    "0x0d21DA3ce8c2E527C8557f5313f896BF2097EbB9",
-    "0xb7e85A365d69A145C5F4e83843656511755cDFB4",
-    "0x9FFB50c414faFaE03Fab346Af0f8ec1e53663170",
-    "0x6C213E21e9bf0D769C80FD2acEfa5ae25313533F",
-    "0xd22807714d0e201f9321AE9360aD75022E2e085C",
-    "0x9565332E63Ac632Bc196626A85baC365b40bC5B7",
-    "0x287c567A65049Db95BE62D054BE8523737d98ba3",
-    "0x5Bd2070c6dA48b341E95bB1eD687FC9f1c864209"
+    "null"
```
Comment (ezynda3) [inline deployments/goerli.diamond.staging.json:3]: This can be removed.

File: scripts/make-diamond-immutable.sh [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  NETWORK=$(cat ./networks | gum filter --placeholder "Network...")
+
+  # define path of JSON file to get diamond address from
+	ADDRS="deployments/$NETWORK.${FILE_SUFFIX}json"
+
+  # get diamond address from path
+  DIAMOND=$(jq -r '.LiFiDiamondImmutable' $ADDRS)
+
+
+  echo "!!! ATTENTION !!!"
```
Comment (ezynda3) [inline scripts/make-diamond-immutable.sh:24]: Not required but you can use some cool `gum` styling to make this message stand out 

e.g. from the docs
```
gum style \
	--foreground 212 --border-foreground 212 --border double \
	--align center --width 50 --margin "1 2" --padding "2 4" \
	'Bubble Gum (1¢)' 'So sweet and so fresh!'
```

Comment (DanTheDev-il) [reply scripts/make-diamond-immutable.sh]: applied

File: deployments/goerli.diamond.staging.json (hunk 1 above)
Comment (DanTheDev-il) [reply deployments/goerli.diamond.staging.json]: done

Comment (0xDEnYO) [reply deployments/goerli.diamond.staging.json]: removed

File: scripts/make-diamond-immutable.sh (hunk 2 above)
Comment (0xDEnYO) [reply scripts/make-diamond-immutable.sh]: applied

**Pull request merged**